src/core/test_helpers/kbn_server.ts

```diff
@@ -140,7 +140,7 @@ export interface TestElasticsearchServer {
   start: (esArgs: string[], esEnvVars: Record<string, string>) => Promise<void>;
   stop: () => Promise<void>;
   cleanup: () => Promise<void>;
-  getClient: () => Client;
+  getClient: () => KibanaClient;
```

Comment:
incorrect typings. `getClients` already returns the new ES client. https://github.com/elastic/kibana/blob/dd10c8b5f2197c936a2efbc45feed7fbb88ebe08/packages/kbn-test/src/legacy_es/legacy_es_test_cluster.js#L105